mne/time_frequency/csd.py

```diff
@@ -255,3 +255,162 @@ def compute_epochs_csd(epochs, mode='multitaper', fmin=0, fmax=np.inf,
                                              frequencies=frequencies[i],
                                              n_fft=n_fft))
         return csds
+
+@verbose
+def csd_array(X, sfreq, mode='multitaper', fmin=0, fmax=np.inf,
+              fsum=True, n_fft=None, mt_bandwidth=None,
+              mt_adaptive=False, mt_low_bias=True, verbose=None):
+    """Estimate cross-spectral density from n_trials x n_series x n_times
```

Comment:
I'm addressing this now. For this function I personally like the shapes called something like `n_replicates x n_series x n_times` since the function is not restricted to operating on `Epochs`. What do you guys think?